deploy/quickstart.sh

```diff
@@ -7,17 +7,26 @@ set -e
 #  Upstream containers: ./quickstart.sh
 #  Downstream containers: ./quickstart.sh --downstream-secret=~/6340056-cloudops-pull-secret.yaml
 oc login -u system:admin
-oc new-project sa-telemetry
+#if [ -z "$(oc projects | grep -o 'sa-telemetry')" ]; then #project does not exist
+if oc get project sa-telemetry; then 
```

Comment:
This will fail if the project exists, but you're not already on the `sa-telemetry` project. You need to explicitly change to the `sa-telemetry` project, or at least make sure it's selected.

Reply:
My more recent commits take care of this

Reply:
That is a good point